web/app.js

```diff
@@ -838,9 +838,7 @@ const PDFViewerApplication = {
         this.open({ url, length, originalUrl });
       },
       onError: err => {
-        this.l10n.get("pdfjs-loading-error").then(msg => {
-          this._documentError(msg, err);
-        });
+        this._documentError("pdfjs-loading-error", err);
```

Comment:
Given that we didn't previously block, in which case this method would have *returned* the result of `this.l10n.get(...)`, I cannot really see why that's necessary now!?

The other call-site is slightly "special", since we want to re-throw the original error, however I've updated that code to remove the `async` part.